test/src/test/java/org/axonframework/test/aggregate/FixtureTestNonEventSourcing.java

```diff
@@ -0,0 +1,36 @@
+package org.axonframework.test.aggregate;
+
+import java.util.UUID;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class FixtureTestNonEventSourcing {
```

Comment:
The build server is configured to run tests for classes ending on `Test` or containing `Test_`. Please rename to `FixtureTest_NonEventSourcing`